### PR TITLE
Flip card animation added

### LINKED ISSUE #266: Animation to cards on landing page
## 🚀 Feature

The cards which represent music box,The band, piano....I would like to add the flip card animation to it.
Please assign me under lgmsoc.

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
### Description

Flip card animation added to the cards.

### Checklist

- [x] I am making a proper pull request, not a spam.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

(Write your answer here.)
Fixes #266 

## Add relevant screenshot or video (if any)

(Add here)
Untitled

### CODE REVIEW

**Comment by nilisha-jais** [comment]
Hey @Khushi-agarwal you have not implemented the feature completely (missing in the games and dance section). Please check it

**Comment by Khushi-agarwal** [comment]
> Hey @Khushi-agarwal you have not implemented the feature completely (missing in the games and dance section). Please check it

@nilisha-jais it's added now.

File: index.html [hunk 1]
```diff
@@ -250,13 +357,23 @@ <h3>Can You Recognise Your Favourite Singer?</h3>
...
-          <span class="girl">💃</span>
+          <div class="flip-card">
+            <div class="flip-card-inner">
+              <div class="flip-card-front">
+                <h1>Lets Dance</h1>
+                
+               </div>
+              <div class="flip-card-back">
+                <span><i class="bx bx-tada bxs-medal"></i></span>
+                <h3>Test your memory by this music game</h3>
```
Comment (nilisha-jais) [inline index.html]: You have mistakenly added this in the dance section. Please fix

Comment (Khushi-agarwal) [reply index.html]: @nilisha-jais sorry for the mistake.Its rectified now.

**Comment by nilisha-jais** [comment]
Screenshot (38)
@Khushi-agarwal have'nt you setup your git bash with the same mail id as in your GitHub account?
You are more likely to face an issue. Your name might not appear in the Contributors list

**Comment by Khushi-agarwal** [comment]
> Screenshot (38)
> @Khushi-agarwal have'nt you setup your git bash with the same mail id as in your GitHub account?
> You are more likely to face an issue. Your name might not appear in the Contributors list

No actually it's with a different id.I hope that might not affect my contributions.

**Comment by Khushi-agarwal** [comment]
@nilisha-jais @niloysikdar any other changes required.

**Comment by nilisha-jais** [comment]
> > Screenshot (38)
> > @Khushi-agarwal have'nt you setup your git bash with the same mail id as in your GitHub account?
> > You are more likely to face an issue. Your name might not appear in the Contributors list
> 
> No actually it's with a different id.I hope that might not affect my contributions.

Please fix this then. Otherwise you won't appear as a contributor and even we would be unable to help

**Comment by Khushi-agarwal** [comment]
> > > Screenshot (38)
> > > @Khushi-agarwal have'nt you setup your git bash with the same mail id as in your GitHub account?
> > > You are more likely to face an issue. Your name might not appear in the Contributors list
> > 
> > 
> > No actually it's with a different id.I hope that might not affect my contributions.
> 
> Please fix this then. Otherwise you won't appear as a contributor and even we would be unable to help

I have fixed it but since this pr was made earlier so here the changes are not reflected. I will take care of it from the next pr. Thank you for telling about it:)

**Comment by Khushi-agarwal** [comment]
@nilisha-jais Its fixed now. Now my name is showing and also I have resolved all the conflicts.

**Comment by nilisha-jais** [comment]
Great! Thank you:)

**Review by nilisha-jais — APPROVED** [review]

**Pull request merged**